app/controllers/posts_controller.rb

```diff
@@ -1,25 +1,40 @@
 class PostsController < ApplicationController
   def index
-    @posts = Post.where(author_id: params[:user_id])
-  end
```

Comment:
- [ ] KIndly note that you were supposed to fix the N+1 problems when fetching the posts and the associated comments. Kindly refer to [this](https://dev.to/junko911/rails-n-1-queries-and-eager-loading-10eh) article provided in the activity. It could be something like included below:

```
@user = User.find(params[:user_id])
    @userposts = @user.posts.includes(:comments)
```